### PR TITLE
Update mentors.yml

### PR DESCRIPTION
**Pull request by chartgerink**
Just some minor additions :-) I added German because I'm learning and can understand a reasonable amount.

### CODE REVIEW

File: _data/round-4/mentors.yml [hunk 1]
```diff
@@ -154,9 +154,9 @@ Technovation Regional ambassador"
   location: Tilburg, The Netherlands
   github: chartgerink
   twitter: chartgerink
-  bio: "PhD-candidate in methods and statistics, focusing on meta-research."
+  bio: "PhD-candidate in methods and statistics, focusing on meta-research and in particular detecting data fabrication. Shifting focus now towards incentivizing valid science practices based on which researchers can actually be evaluated properly. Enthousiastic about loads of things!"
```
Comment (betatim) [inline _data/round-4/mentors.yml]: Enthousiastic -> Enthusiastic or is it a word play I don't get?

**Comment by betatim** [comment]
Good to merge after minor comment

**Comment by chartgerink** [comment]
Ah damn, confabulated it with the Dutch "enthousiast" 🤕

**Comment by acabunoc** [comment]
👍 thanks @chartgerink & @betatim 🎉 🎉 🎉

File: _data/round-4/mentors.yml [hunk 2]
```diff
@@ -154,7 +154,7 @@ Technovation Regional ambassador"
   location: Tilburg, The Netherlands
   github: chartgerink
   twitter: chartgerink
-  bio: "PhD-candidate in methods and statistics, focusing on meta-research and in particular detecting data fabrication. Shifting focus now towards incentivizing valid science practices based on which researchers can actually be evaluated properly. Enthousiastic about loads of things!"
```
Comment (acabunoc) [inline _data/round-4/mentors.yml]: languages are hard!

**Pull request merged**